src/sentry/notifications/utils/participants.py

```diff
@@ -261,6 +262,21 @@ def disabled_users_from_project(project: Project) -> Mapping[ExternalProviders,
     return output
 
 
+def get_suspect_committers(project: Project, event: Event) -> List[APIUser]:
```

Comment:
Thank you, I renamed the function.

I didn't choose to move the function, since converting the JSON returned from `get_serialized_event_file_committers` to type `APIUser` is specific to `participants.py`. Let me know if you disagree with this logic, as I'm not sure either.